Sources/Subs-Attachments.php

```diff
@@ -1420,4 +1457,125 @@ function prepareAttachsByMsg($msgIDs)
 	}
 }
 
+/**
+ * Gets the dimensions of an SVG image (specifically, of its viewport).
+ *
+ * See https://www.w3.org/TR/SVG11/coords.html#IntrinsicSizing
+ *
+ * @param string $filepath The path to the SVG file.
+ * @return array The width and height of the SVG image in pixels.
+ */
+function getSvgSize($filepath)
```

Comment:
Should this function not be moved to Subs-Graphics?

Reply:
Good idea.